### DISCUSSION TITLE
Download

### DISCUSSION THREAD

**Discussion by SeyDow** [upptime/upptime#747] [Q&A] [answered]
How do i download this ? Can u explain me ? :) Thank you ^^

**Comment by SeyDow** [comment]
Tell me the entier procedure to have the injector for the skins thank you a lot :)

**Comment by AnandChowdhary** [comment, marked as answer]
You don't need to download or install anything. Here's the getting started guide for using Upptime: https://upptime.js.org/docs/get-started
